Gemfile

```diff
@@ -6,7 +6,7 @@ ruby '2.3.1'
 gem 'foreman'
 
 # web server
-gem 'unicorn'
+gem 'puma'
```

Comment:
I'm wondering wether we should specify versions for all the gems.

I normally always set a pessimistic lock (`~>`), but I understand why we might prefer to rely on the `gemfile.lock` for this.

Still, it looks like in this case the bundle was locked with an outdated version (3.4.0) rather than the current one (3.6.0).